uninstaller/uninstall-helper.ps1

```diff
@@ -72,7 +72,12 @@ $pathFolders | foreach {
 [string] $delimitedFolders = $folderList -Join ';'
 [Environment]::SetEnvironmentVariable("Path", $delimitedFolders, "User")
 
-Remove-Item -Path "HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\DevelopmentSuite$timeStamp"
+for ($i=0; $i -le $key.Count; $i++) {
```

Comment:
Where is $key variable coming from?

Reply:
ooh I  forgot to write `$key = Get-ChildItem "HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall" -Name`  code